contracts/IdentityFactory.sol

```diff
@@ -0,0 +1,36 @@
+import "./CloneFactory.sol";
+import "./Identity.sol";
+
+pragma solidity 0.6.2;
+
+contract IdentityFactory is CloneFactory {
+    address public libraryAddress;
+
+    Identity[] public identityAddresses;
```

Comment:
Do we really need to store the created identities in an array? Gas price will increase with time so it might eventually be an issue.
I think the `IdentityCreated` event is enough to compute the list of created identities off-chain.

Reply:
Not sure it will be a further issue. If it is the case, we can reissue a new factory?

What about replacing with a mapping (owner => identity)? 

The idea behind this is to have all identities issued by the factory by calling the contract's function

Reply:
But do we need to have the list of all identities issued by the factory?

Reply:
@Joachim-Lebrun ?

Reply:
i agree with Kevin, it's not useful as we can get it offchain with the events `IdentityCreated` no need to store this array on the BC imo